### DISCUSSION TITLE
دیت پیکر پلاگین ها

### DISCUSSION THREAD

**Discussion by JaVAD-NX** [wordpress-parsi/wp-parsidate#212] [General]
چجوری میتونم برای دیت پیکر پلاگین های دیگه تقویم شمسی  رو بیارم؟

**Comment by man4toman** [comment]
سلام خیلی بستگی داره چه افزونه‌ای مدنظرتون باشه.
براساس اینکه افزونه به چه صورت داره از انتخابگر تاریخ استفاه میکنه، باید روشش رو اجرایی کرد.
